opta/commands/output.py

```diff
@@ -14,10 +15,19 @@
 @click.option(
     "-e", "--env", default=None, help="The env to use when loading the config file"
 )
-def output(config: str, env: Optional[str],) -> None:
+@click.option(
+    "--local",
+    is_flag=True,
+    default=False,
+    help="""Use the local Kubernetes cluster for development and testing, irrespective of the environment specified inside the opta service yaml file""",
+    hidden=False,
+)
```

Comment:
it looks like it's possible to create a option once and reuse it in multiple commands: https://github.com/Carbyne/NOG5Thesis/blob/b78b03ac9ba337eabc3e3c3d479a4edd07260f49/netoglyc5/nog5/nog5/cli.py#L17